modules/admin_manual/pages/configuration/server/security/hsmdaemon.adoc

```diff
@@ -56,6 +56,10 @@ HSM support consists of two core parts:
 
 We recommend running hsmdaemon on every web server to reduce latency. 
 
+== Getting the hsmdaemon?
```

Comment:
Using a questionmark does not look good (imho)

Reply:
Will remove that.